GetandTakeAPI/Startup/Extensions/AppConfigurationExtensions.cs

```diff
@@ -0,0 +1,39 @@
+﻿using GetandTake.Configuration.Extensions;
+using GetandTake.Configuration.Settings;
+
+namespace GetandTakeAPI.Startup.Extensions;
+
+public static class AppConfigurationExtensions
+{
+    public static void Configure(this WebApplication app, AppSettings appSettings)
+    {
+        // Configure the HTTP request pipeline.
+        if (app.Environment.IsDevelopment())
+        {
+            app.UseSwagger();
+            app.UseSwaggerUI();
+        }
+
+        app.UseCors(x => x
+            .SetIsOriginAllowed(_ => true)
+            .AllowAnyHeader()
+            .AllowAnyMethod()
+            .AllowCredentials()
+        );
+
+        app.UseStaticFiles();
+
+        //app.RegisterExceptionHandler();
```

Comment:
Don't forget on of the most important rules: never leave committed commented code!
If you are 100% sure that code is needed, you should add a comment to specify reason of comment, otherwise, always remove such code!